### DISCUSSION TITLE
Changing pane orientation?

### DISCUSSION THREAD

**Discussion by refi64** [zellij-org/zellij#1931] [Q&A]
Hello! I've been playing with using ZelliJ for a while now, and it's great, but there's something I'm a bit stuck on.

Let's say I were to open two panes side-by-side. Then, I decide that I want to switch them to be one on top and the other on the bottom. Right now, I can do that by opening some new unused panes, shuffling them around, and then closing the unused ones. This...works, but it's a bit cumbersome. Does ZelliJ have any built-in way to do this, or maybe at least some way to make it easier?

**Comment by imsnif** [comment]
Not yet, but hopefully very soon. We need to get through this piece of technical debt before: https://github.com/zellij-org/zellij/issues/742
